Source/Core/DiscIO/NANDContentLoader.cpp

```diff
@@ -190,8 +190,14 @@ void CNANDContentLoader::InitializeContentEntries(const std::vector<u8>& tmd, co
 		content.m_Index     = Common::swap16(&tmd[entry_offset + 0x01E8]);
 		content.m_Type      = Common::swap16(&tmd[entry_offset + 0x01EA]);
 		content.m_Size      = static_cast<u32>(Common::swap64(&tmd[entry_offset + 0x01EC]));
-		std::copy(&tmd[entry_offset + 0x01E4], &tmd[entry_offset + 0x01E4 + 36], content.m_Header);
-		std::copy(&tmd[entry_offset + 0x01F4], &tmd[entry_offset + 0x01F4 + 20], content.m_SHA1Hash);
+
+		const auto header_begin = std::next(tmd.begin(), entry_offset + 0x01E4);
+		const auto header_end   = std::next(header_begin, sizeof(content.m_Header));
```

Comment:
Please use the constexpr ArraySize function in CommonFuncs. It'll correctly provide the element size even if it's switched to a different data type for whatever reason.